docs/sharding.md

```diff
@@ -85,4 +84,35 @@ To start WildBeast in shard mode, run the startup command as described above. Ex
 node DougBot.js --shardmode --shardcount=10 --shardid=0
 ```
 
+Repeat this command until 10 shard processes are running. Using something like a PM2 process list in JSON format is recommended. Example (File is processes.json):
```

Comment:
While this could work, I'd put `"force": true` for everything to prevent PM2 from complaining about starting the same process multiple times.